### PR TITLE
Updates helm charts and fix configs

### PR DESCRIPTION
**Pull request by rish691**

### CODE REVIEW

File: kubernetes/hypertrace.sh [hunk 1]
```diff
@@ -166,13 +166,13 @@ case "$subcommand" in
       mkdir ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/pre-install-tasks/
       create_helm_templates platform-services ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/manifests.yaml
       echo "[INFO] creating helm deployment template for pre-install tasks"
-      python pre_post_install_task_generator.py pre-install ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/manifests.yaml ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/pre-install-tasks/pre-install-manifests.yaml
+      python3 pre_post_install_task_generator.py pre-install ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/manifests.yaml ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/pre-install-tasks/pre-install-manifests.yaml
```
Comment (kotharironak) [inline kubernetes/hypertrace.sh:169]: Can you keep this to `python` only?

File: kubernetes/hypertrace.sh [hunk 2]
```diff
@@ -166,13 +166,13 @@ case "$subcommand" in
...
       echo "[INFO] pre-install task manifests are generated at: " ${HYPERTRACE_HOME}"/platform-services/helm-deployment-templates/pre-install-tasks/pre-install-manifests.yaml"
     elif [[ "$OPTION" == "--deps" && "$OPTION_ARG" == "post-install-tasks" ]]; then
       mkdir ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/post-install-tasks/
       create_helm_templates platform-services ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/manifests.yaml
       echo "[INFO] creating helm deployment template for post-install tasks"
-      python pre_post_install_task_generator.py post-install ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/manifests.yaml ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/post-install-tasks/post-install-manifests.yaml
+      python3 pre_post_install_task_generator.py post-install ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/manifests.yaml ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/post-install-tasks/post-install-manifests.yaml
```
Comment (kotharironak) [inline kubernetes/hypertrace.sh:175]: Can you keep this to `python` only?

**Review by kotharironak — COMMENTED** [review]
So, we are not upgrading any data services - pinot, kafka in this release, right?

File: kubernetes/hypertrace.sh (hunk 1 above)
Comment (rish691) [reply kubernetes/hypertrace.sh]: updated

**Comment by rish691** [comment]
> So, we are not upgrading any data services - pinot, kafka in this release, right?

Only pinot has changed, we are skipping that now.

**Comment by kotharironak** [comment]
I also verified locally. It's working fine.
Screenshot 2021-12-16 at 5 04 40 PM


Screenshot 2021-12-16 at 5 06 17 PM
Screenshot 2021-12-16 at 5 06 01 PM

**Review by kotharironak — APPROVED** [review]

**Pull request merged**